stdlib/int64.gr

```diff
@@ -281,18 +520,37 @@ export let rec gte = (x: Int64, y: Int64) => {
   ret
 }
 
-// Int64 bit/logical operations
+/**
+ * @section Bitwise Logic: Boolean operations on the bits of Int64 values.
+ */
+
+/**
+ * Computes the bitwise logical NOT of the given value.
+ *
+ * @param value: The given value
+ * @returns The bitwise logical "not" of the given value.
+ * 
+ * @since v0.2.0
+ */
 @disableGC
-export let rec lnot = (x: Int64) => {
-  let xv = WasmI64.load(WasmI32.fromGrain(x), 8n)
+export let rec lnot = (value: Int64) => {
+  let xv = WasmI64.load(WasmI32.fromGrain(value), 8n)
   let ptr = newInt64(WasmI64.xor(xv, 0xffffffffffffffffN))
   let ret = WasmI32.toGrain(ptr): Int64
-  Memory.decRef(WasmI32.fromGrain(x))
+  Memory.decRef(WasmI32.fromGrain(value))
   Memory.decRef(WasmI32.fromGrain(lnot))
   ret
 }
 
-
+/**
+ * Computes the bitwise logical AND on the given operands.
+ *
+ * @param x: The first operand
+ * @param y: The second operand
+ * @returns The bitwise logical "and" of the given operands
```

Comment:
```suggestion
 * @returns The mask of the given operands
```
Wikipedia says this is "masking"?

Otherwise, we could do something like (similar to MDN):
```suggestion
 * @returns Containing a `1` in each bit position for which the corresponding bits of both operands are `1`
```

Reply:
I like the second one 👍